_posts/2024-02-06-unbunching.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Don't get your buses in a bunch
+author: 2TallTyler
+---
+
+Have you ever waited for a late bus, and then two arrived at once?
+
+This is called "[bus bunching](https://en.wikipedia.org/wiki/Bus_bunching)," and I bet you've seen it happen in OpenTTD -- and not just with buses.
+
+For years, players have requested an easier way to keep vehicles spaced out along their route. In OpenTTD 14, we've added one.
+
+<!-- more -->
+
+To keep vehicles spaced evenly, we can either speed up or slow down vehicles.
+Speeding up vehicles is tricky even in real life, requiring adding extra vehicles, skipping stops, or turning a vehicle around before its final destination.
+It's much easier to simply slow them down. Real transit schedules have extra time built in to pad the schedule, often coinciding with the driver's breaks.
+A late-running vehicle can take a shorter break and depart on time. This is also possible in OpenTTD using timetables, but that's a lot of work.
+The most popular fork of OpenTTD, JGR's Patch Pack, includes an "autoseparation" feature that uses timetables automatically to keep vehicles spread out.
+
+With any delay-based unbunching method, there is a fundamental danger of the feature: deadlocks.
+Vehicles get out of order, they block each other waiting to be on time, and the player has no idea why.
+We've seen a few attempts at solving the problem, most with a relatively complex algorithm to avoid deadlocks, which are hard to test.
+
+## A new approach
+
+There is one place where vehicles cannot deadlock: Depots.
+
+Vehicles in depots don't exist in the game world, so they cannot block each other.
+This is the trick to the new unbunching feature in OpenTTD 14.
+
+![The depot menu, with its new Unbunch action]({% link /static/img/post_2024-02-06-bus-bunching/unbunch.png %})
+
+Depots gain a new action type marking them as the unbunching point for a vehicle, chosen from the depot action dropdown where you'd choose "Service," "Stop," etc.
```

Comment:
I suspect some players won't know vehicles can have orders to visit a depot! Also probably need to mention earlier that this works for shared orders. Perhaps "Vehicles can have orders to get serviced or stop at a depot. This feature adds a new type of depot order, marking that depot as the unbunching point for a group of vehicles which share orders. This is chosen from the ...."